src/lib/test.ts

```diff
@@ -0,0 +1,59 @@
+import { TypeMap } from '../snarky/transaction-leaves.js';
```

Comment:
Also this one?

Reply:
oops, this was just a "scratchpad" file that I used to try how/if types worked, shouldn't have been committed